nixnet/_session/frames.py

```diff
@@ -177,13 +278,30 @@ def __repr__(self):
     def write_bytes(
             self,
             frame_bytes):
-        "http://zone.ni.com/reference/en-XX/help/372841N-01/nixnet/nxwriteframe/"
+        """Write a list of raw bytes (frame data).
+
+        The raw bytes encode one or more frames using the Raw Frame Format.
+
+        Args:
+            frame_bytes: List of bytes, representing frames to transmit.
+
+        Returns:
+            None
+        """
         _funcs.nx_write_frame(self._handle, bytes(frame_bytes), constants.TIMEOUT_NONE)
 
     def write_raw(
             self,
             raw_frames):
-        "http://zone.ni.com/reference/en-XX/help/372841N-01/nixnet/nxwriteframe/"
+        """Write raw CAN frame data.
+
+        Args:
+            raw_frames: One or more :class:`nixnet.types.RawFrame` objects to be
+                written to the session.
```

Comment:
Why not just say "a list of " rather than "One or more"?

The latter implies we are doing type checking for an individual item